eng/pipelines/templates/steps/java-publishing.yml

```diff
@@ -1,10 +1,11 @@
 parameters:
   ArtifactDirectory: not-specified
   RepositoryUrl: not-specified
-  GroupID: not-specified
-  ArtifactID: not-specified
+  GroupID:
+  ArtifactID:
   BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
   JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
+  StageOnly: false
```

Comment:
Plumbed through a StageOnly property which effectively replaces ShouldPublish and AutoPromote from the old release pipelines. Now StageOnly simply controls whether we release the package.

Thought: Should we consider adding a separate job for the actual publish script which requires approval?

Reply:
>Should we consider adding a separate job for the actual publish script which requires approval?

We intentionally decided to not require approvals for the partner release pipelines to allow them to be self-serve. As long as they have access to queue a build the essentially have access/approval for releasing it.

Reply:
All good then!